ignore-zone.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @function steal-socket.ignore-zone ignore-zone
```

Comment:
Do you want may be to move the delay-io examples to the main doc page of steal-socket and remove the child docs hiding both `ignore-zone` and `ignore-ssr`?